roles/variables/defaults/main.yml

```diff
@@ -1675,13 +1676,13 @@ kafka_connect_replicator_monitoring_interceptor_ldap_user: "{{kafka_connect_repl
 ### Password for kafka_connect_replicator_monitoring_interceptor_ldap_user LDAP User
 kafka_connect_replicator_monitoring_interceptor_ldap_password: "{{kafka_connect_replicator_ldap_password}}"
 
-### OAuth User for Confluent Replicator Monitoring Interceptor to authenticate as
+### OAuth Client Id for Confluent Replicator Monitoring Interceptor to authenticate as
 kafka_connect_replicator_monitoring_interceptor_oauth_user: "{{ kafka_connect_replicator_oauth_user }}"
 
-### Password for kafka_connect_replicator_monitoring_interceptor_oauth_user OAuth User
+### Client Secret for kafka_connect_replicator_monitoring_interceptor_oauth_user OAuth User
 kafka_connect_replicator_monitoring_interceptor_oauth_password: "{{ kafka_connect_replicator_oauth_password }}"
 
-### Service principal for kafka_connect_monitoring_interceptor_replicator client in Idp server.
+### Service principal for kafka_connect_monitoring_interceptor_replicator client in IdPserver. Defaults to Connect Replicator Monitoring Interceptor Client Id
```

Comment:
```suggestion
### Service principal for kafka_connect_replicator_monitoring_interceptor_replicator client in IdPserver. Defaults to Connect Replicator Monitoring Interceptor Client Id
```

Reply:
fixed this in other PR which adds auth_mode